ouroboros-consensus/ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Ledger/Block/BFT.hs

```diff
@@ -112,34 +114,28 @@ instance ( SimpleCrypto c
   Forging
 -------------------------------------------------------------------------------}
 
+type instance CannotForge (SimpleBftBlock c c') = Void
+
+type instance ForgeStateInfo (SimpleBftBlock c c') = ()
```

Comment:
In the future, this will only be used when the forge state was updated. So we can use `Void`, as there is no forge state to update. Same for `ForgeStateUpdateFailed` (or what's it called).

Reply:
I decided to return the `ForgeStateInfo` when the forge state was unchanged, so `Void` won't fly.